src/release/breaking-changes/key-event-migration.md

```diff
@@ -18,12 +18,14 @@ In the original key event subsystem, handling each platform's quirks in the
 framework and in client apps caused overly complex code, and the old system
 didn't properly represent the true state of key events on the system.
 
-[RawKeyboard] is the legacy API, and has been deprecated and will be removed in
-the future. The replacements are [HardwareKeyboard] and [KeyEvent] APIs (such as
-[FocusNode.onKeyEvent]).
+[`RawKeyboard`][] is the legacy API, and has been deprecated and
+will be removed in the future.
+The replacements are [`HardwareKeyboard`][] and [`KeyEvent`][] APIs
+(such as [`FocusNode.onKeyEvent`][]).
```

Comment:
**chore:** Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
The legacy API [`RawKeyboard`][] has been deprecated and will be removed in the future.
The [`HardwareKeyboard`][] and [`KeyEvent`][] APIs replace this legacy API.
An example would be [`FocusNode.onKeyEvent`][].
```